### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 18

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#290] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152320140-cdbdf214-af46-4dd3-9857-b96a6f6e8dd9.png)
![image](https://user-images.githubusercontent.com/69389461/152320163-9bb0e4ba-7234-4819-93c4-5b3c8875b059.png)

**Comment by LuginiAndrea** [comment]
1) Vero: $$a R^{\cap} b \wedge b R^{\cap} c \Longrightarrow aRb \wedge bRc, aSb \wedge bSc \Longrightarrow aRc, aSc \Longrightarrow a R^{\cap} c$$
2) Non lo è in quanto non transitiva: Ipotizziamo $$\left(a,b\right) \in R \land \left(b, c\right) \in S \land \nexists \left(a,c\right) \in R \land \nexists \left(a,c\right) \in S \implies \exists \left(a,b\right) \in R^* \land \left(b,c\right) \in R^* \land \left(a,c\right) \notin R^*$$
3) $S - R$ non può essere riflessiva in quanto $\forall a \in A, aSa \wedge aRa$, quindi per assurdo se esistesse la coppia $\left(a,a\right)$ vorrebbe dire che non vale $aRa$, impossibile in quanto $R$ ordine
